pkg/virt-controller/watch/clone/clone_test.go

```diff
@@ -61,18 +61,16 @@ const (
 
 var _ = Describe("Clone", func() {
 	var (
-		ctrl        *gomock.Controller
```

Comment:
How do we achieve "Remove tests/util pkg dependency " in e7dae50d1ba97296eeb8b5427849c24b4c2b23ab ?

Reply:
Oh... I had already handled it in https://github.com/kubevirt/kubevirt/commit/087f26e92691f2e8a7c18be38f0f79f405340297 ad during rebase I haven't noticed it. Will reword the commit message. Thank you!

Reply:
Done!